app/javascript/dashboard/components/widgets/conversation/ReplyBox.vue

```diff
@@ -17,6 +17,7 @@ import Banner from 'dashboard/components/ui/Banner.vue';
 import { REPLY_EDITOR_MODES } from 'dashboard/components/widgets/WootWriter/constants';
 import WootMessageEditor from 'dashboard/components/widgets/WootWriter/Editor.vue';
 import WootAudioRecorder from 'dashboard/components/widgets/WootWriter/AudioRecorder.vue';
+import messageFormatterMixin from 'shared/mixins/messageFormatterMixin';
```

Comment:
@fayazara There is some issue with rebase. The `messageFormatterMixin` is already moved to composable https://github.com/chatwoot/chatwoot/pull/9986